src/frontend/components/UI/ActionIcons/index.tsx

```diff
@@ -21,10 +23,14 @@ import { LIBRARY_TOUR_ID } from 'frontend/screens/Library/components/LibraryTour
 
 interface ActionIconsProps {
   'data-tour'?: string
+  showAlphabetFilter?: boolean
+  onToggleAlphabetFilter?: () => void
 }
 
 export default React.memo(function ActionIcons({
-  'data-tour': dataTour
+  'data-tour': dataTour,
+  showAlphabetFilter,
+  onToggleAlphabetFilter
```

Comment:
the other actions are passed using the library context, not with props, lets keep the same pattern of using the library context